README.md

```diff
@@ -10,15 +10,28 @@
 ## How to use?
 
 ```js
-var log = require('ak-logger');
+var log = require('./index');
 
-log.debug("DEBUG LOGS", debug);
+const person = { name: 'Gustavo', age: 23 }
 
-## Result
+// Printing debug log as an object
+log.debug(person, "DEBUG LOGS");
```

Comment:
there is a very small fix here.
As you can see here

https://github.com/arshadkazmi42/ak-logger/blob/master/index.js#L21

first parameter is the title and second is value, you can make that change and update. I think once thats done, it will be ready to merge to master

Reply:
I do not understand @arshadkazmi42 , in line 
https://github.com/arshadkazmi42/ak-logger/blob/master/index.js#L21
First parameter is the `object`, the second the `title` and the third the flag `isStringify`.

Reply:
@gustafsilva oh. yes. you are correct. sorry my bad. this is correct.
Thank you for you contribution. I will merge it right away